docs/dapp/sapphire/gsn.md

```diff
@@ -0,0 +1,78 @@
+---
+description: CLI instructions for deploying GSN on Sapphire Testnet
+---
+# Gas Station Network
+
+[Gas Station Network](https://docs.opengsn.org) is very useful for improving user experiences of many dApps by making their users send transactions without having native tokens for gas. We make GSN functionalities available on Sapphire ParaTime.
+
+### Package Install
+
+Starting with an empty folder, let us install the [Oasis fork](https://github.com/oasislabs/gsn) of the GSN command line tool by using the following commands:
+
+```shell
+npm init -y
+npm install --save-dev @oasislabs/opengsn-cli 
+```
+
```

Comment:
Let's not put any private keys in our docs. Addresses are fine.

```suggestion
Next, we will export our hex-encoded private key for deploying the gas station network as an environment variable:

```sh
export PRIVATE_KEY=0x...
```
```